cli/sync_service/sync_service.go

```diff
@@ -421,7 +421,7 @@ func loadCSS(org string, fileType string, fileObjects []string) error {
 			return errors.New(msgPrinter.Sprintf("unable to read file object %v, error %v", fileName, err))
 		} else {
 
-			defer fileObject.Close()
+			defer cutil.CloseFileLogError(fileObject)
```

Comment:
Please see my comment above'

Reply:
Moved the contents of the for loop to its own function and used `defer cutil.CloseFileLogError(fileObject)` there so there are not multiple files open at a time but the file will still be closed if there is a panic.